assets/src/utils/common.ts

```diff
@@ -134,3 +134,10 @@ export const parseBool = (val: any) => {
   // have to ignore the false searchValue in 'replace'
   return !isNaN(num) ? !!num : !!String(val).toLowerCase().replace('false', '');
 };
+
+export enum contexts {
```

Comment:
this would probably be better in a different file, but not totally sure where, for now it's fine, but it's not really a *util* more of an application context